README.md

```diff
@@ -11,6 +11,47 @@ The [Cluster API](https://github.com/kubernetes-sigs/cluster-api) brings declara
 
 The API itself is shared across multiple cloud providers allowing for true Virtink hybrid deployments of Kubernetes. It is built atop the lessons learned from previous cluster managers such as [kops](https://github.com/kubernetes/kops) and [kubicorn](http://kubicorn.io/).
 
+## Deploy Cluster API Provider Virtink
+
+Install and customize clusterctl.
+
+```shell
+curl -L https://github.com/smartxworks/cluster-api-provider-virtink/releases/download/v0.1.1/setup-clusterctl.sh | sh
+```
+
+Initialize common providers.
+
+```shell
+clusterctl init --infrastructure virtink:v0.1.1
+```
+
+Create workload cluster template.
+
+```shell
+clusterctl generate cluster sample \
```

Comment:
Same here, there's no need to doc about clusterctl commands if they are common routines, leaving a link to cluster-api quick start would be sufficient here.